sdk/tests/tutorials/ibor/test_reference_portfolios.py

```diff
@@ -33,16 +34,26 @@ def setUpClass(cls):
         cls.instrument_loader = InstrumentLoader(cls.instruments_api)
         cls.instrument_ids = cls.instrument_loader.load_instruments()
 
+        cls.id_generator = IdGenerator(default_scope=TestDataUtilities.tutorials_scope)
```

Comment:
So it looks like all codes would be unique, and all in a default_scope, nice

Although do we want to worry about the potential proliferation of data? We could simply set a unique scope for all reference_portfolios tests, and then delete all portfolios in that scope at the end.

Reply:
We are creating new data, but the delete events will logically clear up the entities (in effectiveDate). Having the actual original events followed by the delete event is fine wrt to space (or so I'm told!)